src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -2561,17 +2551,26 @@ private void embedGenericHandle(ref CORINFO_RESOLVED_TOKEN pResolvedToken, bool
                     }
                 }
 
+#if READYTORUN
```

Comment:
R2R was trying to painstakingly work around our fine grained tracking for full AOT - we can just skip that.